tensorflow/contrib/go/graph_test.go

```diff
@@ -0,0 +1,252 @@
+package tensorflow_test
+
+import (
+	"testing"
+
+	tf "github.com/tensorflow/tensorflow/tensorflow/contrib/go"
+)
+
+func TestGraphGeneration(t *testing.T) {
+	graph := tf.NewGraph()
+	input1 := graph.Placeholder("input1", tf.DtInt32, []int64{3}, []string{})
+	input2 := graph.Placeholder("input2", tf.DtInt32, []int64{3}, []string{})
+	_, err := graph.Op("Add", "output", []*tf.GraphNode{input1, input2}, "", nil)
+	if err != nil {
+		t.Error("Problem trying add two tensord, Error:", err)
```

Comment:
yes, maybe the info is a bit redundant. Do you mean just give the error as it? in that case I can change by something like: t.Error(err)?